core/src/Streamly/Internal/Data/Stream/Type.hs

```diff
@@ -538,3 +541,7 @@ concatMapWith
     -> Stream m a
     -> Stream m b
 concatMapWith par f xs = bindWith par xs f
+
+{-# INLINE_NORMAL fuse #-}
+fuse :: a -> a
+fuse = id
```

Comment:
We can use `error "Operation with "fuse" modifier could not be rewritten". So that we get to know any problems because of the operation not fusing or using the modifier on an operation which does not have a rewrite rule to fuse.